### PR TITLE
Using suffix Element/s for IDL attributes referring to elements

### LINKED ISSUE #1732: Using suffix Element/s for IDL attributes referring to elements
[ARIA IDL](https://w3c.github.io/aria/#idl-interface) still doesn't include attributes that refer to elements, but [AOM IDL](https://wicg.github.io/aom/spec/aria-reflection.html#AriaAttributes) doesn't use any suffix for those attributes (e.g. `ariaActiveDescendant` or `ariaDescribedBy`).

However [Chromium implementation](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/dom/aria_relationship_attributes.idl) and the [AOM explainer](https://wicg.github.io/aom/aria-reflection-explainer.html#reflecting-element-references) use suffix `Element` or `Elements` for them (e.g. `ariaActiveDescendantElement` or `ariaDescribedByElements`).

What's the expected naming for these attributes? Thanks!

### PR DESCRIPTION
**Pull request by jnurthen**
Fixes #1732

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1738.html" title="Last updated on Sep 20, 2022, 1:21 AM UTC (fe65c69)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1738/fde8933...fe65c69.html" title="Last updated on Sep 20, 2022, 1:21 AM UTC (fe65c69)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13831,7 +13831,7 @@ <h3>Disambiguation Pattern</h3>
 				<li>If all trusted dictionary sources list a single spelling of a compound word with no spaces or hyphens, only the first letter of the term is capitalized. For example, neither “place-holder” nor “place holder” are considered valid spellings of the term “placeholder,” so <pref>aria-placeholder</pref> becomes <code>ariaPlaceholder</code> with only the P capitalized.</li>
 				<li>There are currently no acronym-based ARIA attributes, but if future attributes include acronym usage, attempt to match existing DOM conventions (e.g. ID becomes Id).</li>
 			</ul>
-		</section>
```
Comment (scottaohara) [inline index.html:13834]: did we lose a closing section here?

Comment (jnurthen) [reply index.html:13834]: Ty

File: index.html [hunk 2]
```diff
@@ -13831,7 +13831,7 @@ <h3>Disambiguation Pattern</h3>
...
+			<p class="note">The specification does not currently define IDL attributes for ARIA content attributes of types ID reference or ID reference list. Naming of these IDL attributes is being <a href="https://github.com/whatwg/html/issues/3515">discussed</a> and may not follow the rules above.</p>
```
Comment (MelSumner) [inline index.html:13834]:
```suggestion
			<p class="note">The specification does not currently define IDL attributes for ARIA content attributes of types ID reference or ID reference list. Naming of these IDL attributes is being <a href="https://github.com/whatwg/html/issues/3515">discussed</a> and may not follow the rules above.</p>
			</section>
```

**Comment by jnurthen** [comment]
closing - overcome by events

**Pull request closed without merging**